test/Core.Test/Services/StripePaymentServiceTests.cs

```diff
@@ -200,14 +200,14 @@ public async void PurchaseOrganizationAsync_Stripe(SutProvider<StripePaymentServ
             s.Customer == "C-1" &&
             s.Expand[0] == "latest_invoice.payment_intent" &&
             s.Metadata[organization.GatewayIdField()] == organization.Id.ToString() &&
-            s.Items.Count == 2
+            s.Items.Count == 0
```

Comment:
This is a continuation of this issue: https://github.com/bitwarden/server/pull/3164#discussion_r1312498355

If a test is failing, you need to ask why. Previously, this test resulted in a call to the `stripeAdapter` with 2 `Item`s. Now, with the same inputs, there are 0 items. Why? What happened to those items? There's a bug somewhere in your refactor.

Reply:
This test was failing because the organization.UseSecretsManager  was not set to true and we are trying to add a secret manager additional Seats and Additional ServiceAccounts. But I have add organization.UseSecretsManager = true; and the count is now 2  (s.Items.Count == 2)

Reply:
Makes sense! Thanks for investigating it further.